src/pwa-buildpack/project-setup/install-dependencies/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Install project dependencies
+---
+
+In the [previous topic], you created the basic directory structure and files for a Magento theme.
+In this topic you will initialize the `package.json` file and install project dependencies for your Magento PWA theme.
+
+## Initialize and define the package configuration file
+
+1. The following command uses `npm` to create your project's `package.json` file using default project values:
+   
+   ``` bash
+   npm init -y
+   ```
+   
+1. Run the following command to install [peregrine] and [React] as a production dependency:
+
+   ``` bash
+   npm install --save @magento/peregrine \
+   react react-dom react-redux react-router-dom redux
```

Comment:
This needs to change to `react react-dom react-redux react-router-dom redux@3`. (In general, we need to retire this little section and put these dependencies in buildpack directly. Otherwise we'll keep having to come back to edit this!)